WordPress/WordPressApi/WordPressComApi.m

```diff
@@ -7,7 +7,7 @@
 #import "NotificationsManager.h"
 #import "WPUserAgent.h"
 
-static NSString *const WordPressComApiClientEndpointURL = @"https://public-api.wordpress.com/rest/v1.1/";
+static NSString *const WordPressComApiClientEndpointURL = @"https://public-api.wordpress.com/rest/";
```

Comment:
This is now addressed - I added methods that (a bit primitively) verify that the version is present in the path.